### PR TITLE
Mage_Rss - DOC block update

### PR DESCRIPTION
**Pull request by sreichel**
### Description (*)

Added/fixed DOCblock. (#416)

### Contribution checklist (*)
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [x] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Review by colinmollenhour — APPROVED** [review]

File: app/code/core/Mage/Rss/data/rss_setup/data-install-1.6.0.0.php [hunk 1]
```diff
@@ -25,7 +25,7 @@
  */
 
 
-/* @var $installer Mage_Core_Model_Resource_Setup */
+/* @var Mage_Core_Model_Resource_Setup $installer */
```
Comment (tmotyl) [inline app/code/core/Mage/Rss/data/rss_setup/data-install-1.6.0.0.php:28]: Double star

Comment (sreichel) [reply app/code/core/Mage/Rss/data/rss_setup/data-install-1.6.0.0.php:28]: @tmotyl can this be fixed later? Did a quick search and found +1.1k occurrences ...

Comment (tmotyl) [reply app/code/core/Mage/Rss/data/rss_setup/data-install-1.6.0.0.php:28]: Sure

Comment (sreichel) [reply app/code/core/Mage/Rss/data/rss_setup/data-install-1.6.0.0.php:28]: Approve? ;)

**Review by kkrieger85 — APPROVED** [review]

**Pull request merged**